File: Hello.py
#!/usr/bin/env python3
# Author: Kyleigh
# Date: 1/14/2018

def get_stuff():
  name = input("What is your name? ")
  age = int(input("How old are you? "))
  return name, age
    
def print_stuff(name, age):
  print("Nice to meet you, " + name + "!")
  if age > 50:
    print("You're too old!")
  else:
    print("You're too young!")

stuff = get_stuff()
print_stuff(stuff[0], stuff[1])

File: Hello.rb
#!/usr/bin/env ruby
# Author: Kyleigh
# Date: 1/21/2018

def get_stuff()
  print("What is your name? ")
  name = gets.chomp
  print("How old are you? ")
  age = gets.chomp.to_i
  return name, age
end

def print_stuff(name, age)
  puts ("Nice to meet you, #{name}!")
  if age > 50
    puts ("You're too old!")
  else
    puts ("You're too young!")
  end
end

stuff = get_stuff()
print_stuff(stuff[0], stuff[1])
